ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -204,16 +202,11 @@ void compact(CompactionDetails details, byte[] bundleReadBuffer) throws IOExcept
   private void getDeprecatedContainers() {
```

Comment:
There is no return of this method, we can update the java doc

Reply:
fixed.